### PR TITLE
Pubspec Flutter + dart sdk version fix

### PR DESCRIPTION
**Pull request by WieFel**

### CODE REVIEW

**Comment by calvintam236** [comment]
@WieFel please add this PR to the changelog.

**Comment by calvintam236** [comment]
@WieFel also bump version to `3.0.1`.

**Review by calvintam236 — DISMISSED** [review]

**Review by calvintam236 — APPROVED** [review]

**Comment by calvintam236** [comment]
@WieFel done. please make a release

**Comment by WieFel** [comment]
Can you please create the release this time? Am not at PC anymore...

File: pubspec.yaml [hunk 1]
```diff
@@ -1,11 +1,11 @@
 name: tinycolor2
 description: Flutter Color manipulation and conversion, ported from JS tinycolor2
-version: 3.0.0
+version: 3.0.1
 homepage: https://github.com/TinyCommunity/tinycolor2
 
 environment:
   sdk: ">=2.17.0 <3.0.0"
-  flutter: ">=3.0.0"
+  flutter: ">=3.0.0 <4.0.0"
```
Comment (calvintam236) [inline pubspec.yaml:8]: https://github.com/TinyCommunity/tinycolor2/actions/runs/3065842779/jobs/4950358056

@WieFel looks like they don’t want upper restraint for flutter packages

**Pull request merged**